repos/spack_repo/builtin/packages/kokkos_fft/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack.package import *
+
+
+class KokkosFft(CMakePackage):
+    """FFT interfaces for Kokkos C++ Performance Portability Programming EcoSystem"""
+
+    homepage = "https://github.com/kokkos/kokkos-fft"
+    url = "https://github.com/kokkos/kokkos-fft/archive/refs/tags/v0.3.0.tar.gz"
+
+    maintainers("cedricchevalier19", "tpadioleau", "yasahi-hpc")
+
+    license("Apache-2.0 WITH LLVM-exception OR MIT", checked_by="cedricchevalier19")
+
+    version("0.3.0", sha256="a13c423775afec5f9f79fa9a23dd6001d3d63bae9f4786b1e0cd3ed65b3993a3")
+
+    variant("fftw", default=True, description="Enable FFTW")
+    variant("cufft", default=False, description="Enable cufft")
+    variant("hipfft", default=False, description="Enable hipfft")
+    variant("onemkl", default=False, description="Enable oneMKL fft")
```

Comment:
Do we allow different device backends at once? (I do not think so)
If not, a `device_backend` variant that lists the available option can be a cleaner API. 

It is farther away from CMake options, but I think it makes more sense to explain that you can select only one host and only one device FFT backends.

Reply:
done in be968b0ee4cb3f0d632b818774310943e51eb991, what do you think ? I kind of like it!